package.json

```diff
@@ -4,7 +4,7 @@
   "description": "Parse and consume binary streams with a neat DSL",
   "main": "index.js",
   "engines": {
-    "node": ">= 0.10.0 <= 5.10.0"
+    "node": ">= 0.10.0 ^5"
```

Comment:
Does this actually work? I updated my previous comment to say `>= 0.10.0 <= 5`. `>= 0.10 <= 5` would be even easier on the eyes.